src/components/avatars/OcAvatarGroup.vue

```diff
@@ -1,28 +1,26 @@
 <template>
-  <div
-    class="oc-avatar-group"
-    :class="{ 'oc-avatar-group-stacked': stacked }"
-    :uk-tooltip="tooltip"
-  >
-    <template v-if="avatars.length > 0">
-      <oc-avatar
-        v-for="avatar in avatars"
-        :key="avatar.username"
-        :src="avatar.avatar"
-        :user-name="avatar.username"
-        :accessible-label="avatar.displayName"
-        :width="30"
-      />
-    </template>
-    <template v-if="links.length > 0">
-      <oc-avatar-link
-        v-for="(link, index) in links"
-        :key="link.name + index"
-        :name="link.name"
-        :accessible-label="link.name"
-      />
-    </template>
-    <oc-avatar-count v-if="isOverlapping" :count="users.length - maxDisplayed" />
+  <div>
+    <div
+      class="oc-avatar-group"
+      :class="{ 'oc-avatar-group-stacked': stacked }"
+      :uk-tooltip="tooltip"
```

Comment:
During rebasing, please make sure that you don't accidentally add the `uk-tooltip` again. ;-) We have `v-oc-tooltip` now for tooltips.